libs/klv/src/main/java/org/codice/alliance/libs/klv/AttributeNameConstants.java

```diff
@@ -46,39 +54,47 @@
 
     public static final String CORNER = "corner-location";
 
-    public static final String SECURITY_CLASSIFICATION = "security-classification";
+    public static final String SECURITY_CLASSIFICATION = Security.CLASSIFICATION;
 
     public static final String CLASSIFYING_COUNTRY_CODING_METHOD =
             "classifying-country-coding-method";
 
-    public static final String CLASSIFYING_COUNTRY = "classifying-country";
+    public static final String CLASSIFYING_COUNTRY = Security.CLASSIFICATION_SYSTEM;
 
     public static final String OBJECT_COUNTRY_CODING_METHOD = "object-country-coding-method";
 
-    public static final String OBJECT_COUNTRY_CODES = "object-country-codes";
+    public static final String OBJECT_COUNTRY_CODES = Location.COUNTRY_CODE;
 
-    public static final String CHECKSUM = "checksum";
+    public static final String CHECKSUM = "klv-packet-checksum";
 
-    public static final String IMAGE_COORDINATE_SYSTEM = "image-coordinate-system";
+    public static final String IMAGE_COORDINATE_SYSTEM = Location.COORDINATE_REFERENCE_SYSTEM_NAME;
 
-    public static final String IMAGE_SOURCE_SENSOR = "image-source-sensor";
+    public static final String IMAGE_SOURCE_SENSOR = Isr.SENSOR_ID;
 
-    public static final String TARGET_WIDTH = "target-width";
+    public static final String TARGET_WIDTH_METERS = "target-width-meters";
 
     public static final String FRAME_CENTER_ELEVATION = "frame-center-elevation";
 
-    public static final String SENSOR_TRUE_ALTITUDE = "sensor-true-altitude";
+    public static final String SENSOR_TRUE_ALTITUDE = Location.ALTITUDE;
 
     public static final String GROUND_RANGE = "ground-range";
```

Comment:
If these are attribute names we should either add them to the taxonomy or use the ext.* naming convention.